lib/find-view-matches.js

```diff
@@ -57,6 +63,10 @@ function attrSelector(attr, value) {
 	].join('');
 }
 
+function prefixSelector(prefix, value) {
+	return value.replace(new RegExp('^\\'+ prefix +'?(.*)$', 'i'), function (match) { return prefix + match });
```

Comment:
Could you update the function to this new approach? So that I can merge your PR.